### ISSUE TITLE
Exploding gradients problem and binary DD release

### ISSUE THREAD

**Issue by id3509** [RingBDStack/SUGAR#5]
1.Could you please give me the binary DD output and release again? The link is dead。
2.I also encountered the problem of exploding gradients, could you please give the details of the training parameters for the NCI1 or NCI109 dataset? Such as learning rate, batch size, hidden layer settings, etc.
I shall be profoundly grateful if you will give me an answer.

**Comment by Suchun-sv** [comment]
Thanks for your attention. 
1. the binary DD: https://pan.baidu.com/s/1mkh7osIplj85QdhEtEdBCw  6m14 

2.The parameters: we use the optuna framework (https://optuna.org/) for hyperparameter tuning on each dataset. The tuning range is 1e-3 to 1e-5 for learning rate, 32 for batch size, 128 for hidden dimension.
